administrator-guides/google-cloud/auto-translate/README.md

```diff
@@ -0,0 +1,42 @@
+# Google Auto Translate
+
+You can use Google's translation engine to translate every message received on a channel.
+
+For that you will need a Google Translation API key, you can find how to get one and pricing for it's usage from the [Official Google API documentation](https://cloud.google.com/translate/pricing).
+
+> Please note, that the Translation API is a paid feature from Google Cloud. If you decide to aquire this service, you will be paying Google and not Rocket.Chat
```

Comment:
I don't think we should add this here. It may cause some confusion.
If we keep it, fix grammar errors:

"Please note that..." -- remove the comma
"acquire" instead of "aquire"